### ISSUE TITLE
How to list columns to be included in reports?

### ISSUE THREAD

**Issue by dtrezubSV** [github/issue-metrics#198]
Hi! Sorry if this is a dumb question, but I cannot get the action to give me any values for the time spent in columns.

I have columns labeled "In Progress", "In Review", "Blocked", "Ready to Merge".

I tried passing the arg as `LABELS_TO_MEASURE: 'in-progress,blocked,code-review,ready-to-merge'` as per the examples. No deal.

I tried also `LABELS_TO_MEASURE: 'In Progress,Blocked,Code Review,Ready to Merge'`, to no avail either.

Using double quotes didn't work either: `LABELS_TO_MEASURE: '"In Progress","Blocked","Code Review","Ready to Merge"'`

The only metric I can get is Time to Close.

What should be the correct way to list the column labels? Thansk!

**Comment by zkoppert** [comment]
Are you talking about [project columns](https://docs.github.com/en/issues/planning-and-tracking-with-projects/customizing-views-in-your-project/customizing-the-board-layout#setting-the-column-field-in-board-layout)? The action is designed to use [issue labels](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/encouraging-helpful-contributions-to-your-project-with-labels) and measure the time spent with those labels. Does that make sense?

**Comment by dtrezubSV** [comment]
Thanks for taking the time to reply. If we could at least auto-assign a label when an issue changes columns, #57 could become a reality, but it's not possible yet with projects v2. Funny how GitHub releases features but Github itself doesn't use them :P

**Comment by zkoppert** [comment]
Yeah we are in an in between right now because the current version of projects can't trigger anything in order to cause an auto-label event. https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#project_column I'll mark this as blocked and we can revisit when the projects API is improved.
